docker/utils/utils.py

```diff
@@ -715,6 +715,27 @@ def create_host_config(binds=None, port_bindings=None, lxc_conf=None,
     return host_config
 
 
+def create_networking_config(endpoints_config=None):
+    networking_config = {}
+
+    if endpoints_config:
+        networking_config["EndpointsConfig"] = endpoints_config
+
+    return networking_config
+
+
+def create_endpoint_config(version,
+                           aliases=None):
```

Comment:
Yep, just anticipating future additions (these things have a tendency to grow), but it's probably unnecessary